.gitlab-ci.yml

```diff
@@ -612,6 +612,17 @@ library:ci-bedrock2:
     NJOBS: "1"
   timeout: 2h
 
+library:ci-ext_lib:
+  extends: .ci-template
+
+library:ci-brick:
+  extends: .ci-template-flambda
+  needs:
+    - build:edge+flambda
+    - library:ci-iris
+    - plugin:ci-elpi_test
+    - library:ci-ext_lib
```

Comment:
I don't think this will work, the dependencies need to use flambda if the job uses flambda (fiat_crypto_ocaml is the exception because it doesn't run coqc)

Reply:
quickchick currently builds extlib in its job so if we add a extlib job we should make quickchick depend on it
quickchick is flambda so extlib should also be flambda